__tests__/app.test.js

```diff
@@ -80,3 +80,43 @@ describe("GET/api/endpoints", () => {
       });
   });
 });
+
+describe("GET/api/articles/article_id", () => {
+  test("returns status:200 and returns all the articles according to article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
```

Comment:
From our data we know exactly what this article should like so we should be asserting those values here not just the keys 👍